ArduPlane/tiltrotor.cpp

```diff
@@ -245,7 +249,7 @@ void Tiltrotor::continuous_update(void)
         }
         if (!quadplane.motor_test.running) {
             // the motors are all the way forward, start using them for fwd thrust
-            const uint16_t mask = is_zero(current_throttle)?0U:tilt_mask.get();
+            const uint16_t mask = tilt_mask.get();
```

Comment:
I think will mean you get spin min in motor test rather than no motors spinning?

Reply:
it is inside a if() checking for !quadplane.motor_test.running, so I don't think it could impact motor test